packages/kcms/src/match/service/createRunResult.ts

```diff
@@ -0,0 +1,63 @@
+import { Option, Result } from '@mikuroxina/mini-fn';
+import { MatchType } from 'config';
+import { SnowflakeIDGenerator } from '../../id/main';
+import { MainMatchID } from '../model/main';
+import { PreMatchID } from '../model/pre';
+import { MainMatchRepository, PreMatchRepository } from '../model/repository';
+import { CreateRunResultArgs, RunResult } from '../model/runResult';
+
+/**
+ * 試合の走行結果を複数更新するService
+ */
+export class CreateRunResultService {
+  constructor(
+    private readonly idGenerator: SnowflakeIDGenerator,
+    private readonly preMatchRepository: PreMatchRepository,
+    private readonly mainMatchRepository: MainMatchRepository
+  ) {}
+
+  /**
+   *  試合の走行結果を複数送信します
```

Comment:
複数なのは引数から自明なのと、送信ではないのでこうすべきかと
```suggestion
   *  試合の走行結果を追加します
```